.github/workflows/dco-check.yml

```
@@ -35,79 +28,133 @@ on:
+  statuses: write
+
 jobs:
   dco:
     name: DCO Sign-Off
     runs-on: ubuntu-latest
-    # Run on pull_request events always; on issue_comment only for PRs with
-    # the confirmation phrase from authorized users
     if: >
-      github.event_name == 'pull_request' ||
+      github.event_name == 'pull_request_target' ||
       (github.event_name == 'issue_comment' &&
```

actionlint (1.7.12)

[error] undefined function "toLower". available functions are "always", "cancelled", "case", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)